src/index.js

```diff
@@ -106,10 +103,8 @@ MyArray.prototype.sort = function(cb) {
 
   if (!callback) {
     callback = function(a, b) {
-      let firstArg = a;
-      let secondArg = b;
-      firstArg = `${firstArg}`;
-      secondArg = `${secondArg}`;
+      const firstArg = `${a}`;
```

Comment:
Why do you need these variables firstArg and secondArg at all? You may just use A and B in your further comparison. You don't use them anywhere else.
What do you think? It would make your code 2 lines shorter ))

Reply:
For string cast of parameters + do not mutate of parameters (eslint rule)